specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -6888,6 +7346,276 @@
+          "type": "string",
+          "description": "Gets or sets the disk name."
+        },
+        "caching": {
+          "$ref": "#/definitions/Caching",
+          "description": "Gets or sets the caching type."
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets the disk size in GB."
+        },
+        "managedDisk": {
+          "$ref": "#/definitions/ManagedDiskParameters",
+          "description": "Gets the managed disk details"
+        },
+        "diskRestorePoint": {
+          "$ref": "#/definitions/ApiEntityReference",
+          "description": "Gets or sets the disk restore point Id."
+        }
+      },
+      "description": "Describes an Operating System disk."
+    },
+    "RestorePointSourceVMDataDisk": {
+      "properties": {
+        "lun": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the logical unit number."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the disk name."
+        },
+        "caching": {
+          "$ref": "#/definitions/Caching",
+          "description": "Gets or sets the caching type."
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the initial disk size in GB for blank data disks, and the new desired size for existing OS and Data disks."
+        },
+        "managedDisk": {
+          "$ref": "#/definitions/ManagedDiskParameters",
+          "description": "Gets the managed disk details"
+        },
+        "diskRestorePoint": {
+          "$ref": "#/definitions/ApiEntityReference",
+          "description": "Gets or sets the disk restore point Id."
+        }
+      },
+      "description": "Describes a data disk."
+    },
+    "RestorePointProvisioningDetails": {
+      "properties": {
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the creation time of the restore point."
+        },
+        "totalUsedSizeInBytes": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets the total size of the data in all the disks which are part of the restore point."
+        },
+        "statusCode": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets the status of the Create restore point operation."
+        },
+        "statusMessage": {
+          "type": "string",
+          "description": "Gets the status message of the Create restore point operation."
+        }
+      },
+      "description": "Restore Point Provisioning details."
+    },
+    "RestorePointCreate": {
+      "properties": {
+        "excludeDisks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiEntityReference"
+          },
+          "readOnly": false,
```

Comment:
Do we need this, what is the default value for this property?

Reply:
sry  can you clarify the question please?
Are you asking if the $ref": "#/definitions/ApiEntityReference" is needed? are you asking if "readOnly": false, is needed?

This is the ExcludeDisk property when creating a restore point.
It has to be a list of ApiEntityRef for the data disks or os disk that the custoemr wishes to exclude.

It is an optional param when creating restore point. there is no default vlaue.

Reply:
yes Rob, if it is not readonly, then you dont need to explicitly specify it.